python/tank/descriptor/io_descriptor/github_release.py

```diff
@@ -304,10 +301,6 @@ def has_remote_access(self):
             # for private repos accessed without a token, so there's no way to helpfully warn the
             # user if they try to download from a private repo.
             can_connect = response_code == 200
-            # @todo Perhaps deal with redirects (which may occur in the case of a
-            # renamed repo) here. The HTTPRedirectHandler may be a good option
-            # for this.
-            # (https://docs.python.org/2/library/urllib2.html#urllib2.HTTPRedirectHandler)
```

Comment:
Are you sure you want to remove the full comment? Maybe we should just update the URL to latest Py3?